allensdk/brain_observatory/vbn_2022/metadata_writer/schemas.py

```diff
@@ -47,6 +47,18 @@ class VBN2022MetadataWriterInputSchema(argschema.ArgSchema):
           "{ecephys_nwb_dir}/{ecephys_nwb_prefix}_{ecephys_session_id}.nwb")
     )
 
+    supplemental_columns = argschema.fields.List(
```

Comment:
Should this rather be an input file? Passing a long list of dicts through the command line would be cumbersome.

Reply:
I don't want to proliferate the number of input files we have to keep track of. These input.jsons are large, but they have the virtue of carrying everything we need in one package.

I also don't see users specifying this field (or, or that matter, `probes_to_skip`) on the command line.

I'd rather leave this as it is.